src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -420,30 +421,55 @@ private void updateSupplementaryEvidence(
         CaseDetails exceptionRecordDetails,
         boolean ignoreWarnings
     ) {
-        Optional<String> attachedToCase = getCaseExceptionRecordIsAttachedTo(
-            callBackEvent.exceptionRecordJurisdiction,
-            callBackEvent.exceptionRecordId
+        CaseDetails fetchedExceptionRecord = ccdApi.getCase(
+            callBackEvent.exceptionRecordId.toString(),
+            callBackEvent.exceptionRecordJurisdiction
```

Comment:
not sure this is correct. You are fetching exception record but that is already passed in the callback ?

Reply:
trying to get existing case